portfolionav.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<link rel="stylesheet" href="portfolio.css">
```

Comment:
Don't forget to link to normalize.css here as well! That should be done just before you link to portfolio.css